multi-ai-cli/backends/mistral_web.py

```
@@ -1,20 +1,76 @@
+            "model": "mistral-large-latest",
+            "messages": messages,
+            "stream": True
+        }
+
+        try:
+            kwargs = {"headers": headers, "json": payload, "cookies": self.cookies, "stream": True}
+            if hasattr(curl_requests, "Session"):
+                s = curl_requests.Session()
+                resp = s.post(self.base_url, impersonate="chrome120", **kwargs)
+            else:
+                resp = curl_requests.post(self.base_url, **kwargs)
```

Ruff (0.16.1)

[error] Probable use of `requests` call without timeout

(S113)